.github/workflows/update-dist.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
 
     permissions:
-      contents: read
```

Comment:
why is this change needed?

Reply:
Otherwise, the action that generates the dist folder fails to push to the repo.
https://github.com/flox/install-flox-action/pull/98/commits/24ff7b6d68759227cff22a53ddd75663c9c3a6ed